yarn-completion.plugin.zsh

```diff
@@ -81,9 +81,6 @@ _yc_yarn_remove_completion() {
 }
 
 _yc_yarn_run_completion() {
-  # Only run on `yarn run ?`
-  [[ ! "$(_yc_no_of_yarn_args)" = "3" ]] && return
-
```

Comment:
I've removed this because with yarn, you don't have to write `yarn run <cmd>` but `yarn <cmd>`